### ISSUE TITLE
Android Cordova not showing map.

### ISSUE THREAD

**Issue by Kestanous** [pauloborges/meteor-mapbox#3]
I can get the map to load on the web fine. Including on the default Chome app on an Android device. But when I run it as a native app on an Android it gives me a white screen with no map controls. I can get it to work on the iOS emulator and will be testing on an iPhone soon. I thought maybe it was a css issue but I can't see how:

html:
```html
<body> <!-- final result after iron:router, before MapBox init -->
<div id='mapDisplay'>
  <div class="mapper" id="map"></div>
</div>
</body>
```
css:
```css
div#mapDisplay {
  height: 100%;
  width: 100%;
 }
div.mapper {
  width: 100%;
  height: 100%;
}
```

**Comment by Kestanous** [comment]
I am getting the following:

```
Loaded "https://api.tiles.mapbox.com/mapbox.js/v2.1.4/mapbox.js"
Mapbox.loaded()
false
```
The second two are coming from:
```js 
Tracker.autorun(function (c) {
  console.log( 'Mapbox.loaded()' );
  console.log( Mapbox.loaded() );
});
```

**Comment by Kestanous** [comment]
Also worth mentioning, I am not using any plugins:
```js
Meteor.startup(function() {
  Mapbox.debug = true;
  Mapbox.load();
});
```

**Comment by Kestanous** [comment]
It looks like it has to do with the css. The load event is not firing:

```js 
var loadCss = function (href) {
  var elem = document.createElement('link');
  elem.rel = 'stylesheet';
  elem.href = href;
  elem.addEventListener('load', _.partial(onLoadedFile, href), false); //not fired
  ...
```

**Comment by pauloborges** [comment]
I'm trying to reproduce it without success. I successfully installed [this demo](https://github.com/pauloborges/meteor-mapbox-demo) in both android simulator and real device.

Can you provide me a full example?

**Comment by Kestanous** [comment]
I just got it working on my device with the PR I just made. That said I will try your demo. Worth noting that I am using Android 4.2.2 on this device.

**Comment by Kestanous** [comment]
I can reproduce with your demo.

**Comment by Kestanous** [comment]
![screenshot_2015-01-21-14-10-05](https://cloud.githubusercontent.com/assets/1283884/5843012/5758363e-a177-11e4-80f8-967dda6045a6.png)

**Comment by Kestanous** [comment]
Fixes with patch #4
